### PR TITLE
Issue #11637: Resolve shellcheck SC2026

### PR DESCRIPTION
**Pull request by shubh220922**
Related to #11637 
Info referred to: https://www.shellcheck.net/wiki/SC2026

Checking CI.

### CODE REVIEW

**Comment by shubh220922** [comment]
Problem CI showed:

https://github.com/checkstyle/checkstyle/actions/runs/4096900031/jobs/7064875554#step:4:6

**Comment by rnveach** [comment]
Are you familiar with bash to read the code and try to understand what it is doing?
Do you know what the following lines mean? https://github.com/checkstyle/checkstyle/blob/master/.ci/validation.sh#L20-L22

File: .ci/validation.sh [hunk 1]
```diff
@@ -42,7 +42,7 @@ check-missing-pitests)
 
   list=($(cat pom.xml | \
     xmlstarlet sel --ps -N pom="http://maven.apache.org/POM/4.0.0" \
-    -t -v '//pom:profile[./pom:id[contains(text(),'pitest')]]//pom:targetClasses/pom:param'))
+    -t -v '//pom:profile[./pom:id[contains(text(),'"'pitest'"')]]//pom:targetClasses/pom:param'))
```
Comment (nrmancuso) [inline .ci/validation.sh:45]:
```suggestion
    -t -v "//pom:profile[./pom:id[contains(text(),'pitest')]]//pom:targetClasses/pom:param"))
```

Please send temporary commit to show that it works by removing some pitest target class.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

**Comment by shubh220922** [comment]
> Are you familiar with bash to read the code and try to understand what it is doing? Do you know what the following lines mean? https://github.com/checkstyle/checkstyle/blob/master/.ci/validation.sh#L20-L22

(I am writing this reply after doing a complete course on bash (link: https://www.youtube.com/watch?v=e7BufAVwDiM))

I totally didn't realize it was switch case statement there. I am working on it now.

File: .ci/validation.sh (hunk 1 above)
Comment (nrmancuso) [reply .ci/validation.sh:45]: @shubh220922 ping

**Comment by shubh220922** [comment]
Progress of this PR is continued in .

**Pull request closed without merging**